src/kixi/hecuba/weather/service.clj

```diff
@@ -16,6 +18,24 @@
   (with-open [r (io/reader filename)]
     (read (java.io.PushbackReader. r))))
 
+(defn gen-message [entity-id entity-type entity-action]
+  (json/write-str {:entity-id entity-id
+                   :entity-type entity-type
+                   :entity-action entity-action}))
+
+(defn send-to-kafka [args-map entities]
+  (let [zookeeper-brokers (zk/brokers {"zookeeper.connect" (:zookeeper args-map)})
+        kafka-producer (kafka/producer {"metadata.broker.list" (:kafka-brokerlist args-map)
+                                        "serializer.class" "kafka.serializer.DefaultEncoder"
+                                        "partitioner.class" "kafka.producer.DefaultPartitioner"})]
+    (map (fn [entity]
```

Comment:
This probably needs to be wrapped in a doall or replaced with run!

Otherwise the lazy seq won't be realised when you run it as an uberjar.

https://clojuredocs.org/clojure.core/run!